lib/internal/test_runner/tap_lexer.js

```diff
@@ -468,11 +468,19 @@ class TapLexer {
   }
 
   #isLiteralSymbol(char) {
-    return (
-      (char >= 'a' && char <= 'z') ||
-      (char >= 'A' && char <= 'Z') ||
-      this.#isSpecialCharacterSymbol(char)
-    );
+    if (char.charCodeAt(0) > 127) {
+      return true;
+    }
+
+    if (!Number.isNaN(Number(char))) {
+      return false;
+    }
+
+    if (char === "#") {
```

Comment:
Number to number comparisons are faster than string to string comparisons.

If you store `char.charCodeAt(0)` in a variable, you can check `code === 35`

Reply:
thank you :ninja: